src/api/dependenciesConfig.js

```diff
@@ -76,9 +81,8 @@ export type DependencyConfig = {|
   // from the start of the project up until the first specified weightPeriod.
   // If this is set and there are no explicit weightPeriods, this weight will
   // prevail for the whole timeline of the project.
-  // Currently, weight periods aren't yet implemented, so whatever startWeight is
-  // set will apply across all time.
   +startWeight: number,
+  +periods?: $ReadOnlyArray<MintPeriod>,
```

Comment:
What do you think about naming this "additionalPeriods"? It wasn't clear to me that the initial period shouldn't be set here as well

Reply:
We could also remove the `startWeight` setting, and just have users set periods if they want to do any minting. It's a simpler API. What do you think?

Reply:
+1, should simplify to use only periods